libraries/botframework-connector/src/auth/endorsementsValidator.ts

```diff
@@ -24,7 +28,7 @@ export class EndorsementsValidator {
         }
 
         if (endorsements === null) {
-            throw new Error('endorsements required');
+            throw new AuthenticationError('endorsements required', StatusCodes.BAD_REQUEST);
```

Comment:
Unauthorized preferred here. This means that the token was not endorsed for the channel, which is a must and part of the auth validation.